styling-and-appearance/glyphs/common-styles-appearance-glyphs-radglyph.md

```diff
@@ -0,0 +1,169 @@
+
+#### __[XAML] Example 1: Define a RadGlyph in XAML
+
+{{region xaml-common-styling-appearance-glyphs-overview-1}}
+	<telerik:RadGlyph Glyph="&#xe91e;" Foreground="Black" FontSize="100" />
+{{endregion}}
+
+#### __[XAML] Example 2: Use the RadGlyph markup extension
+
+{{region xaml-common-styling-appearance-glyphs-overview-2}}
+	<Image Source="{telerik:RadGlyph Glyph={StaticResource GlyphCs}, Foreground=Black}" />
+{{endregion}}
+
+#### Figure 1: The result from examples 1 and 2
+
+![The C# RadGlyph](images/radglyph-1.png)
+
+> In order to use the glyph references (such as **GlyphCs** in **Example 2**) as static resources in a project, you need to include the **Telerik.Windows.Controls.dll** and merge the **FontResources.xaml** dictionary in the application resources. A list of the available glyphs and their resource keys can be found in the [Glyphs Reference Sheet]({%slug common-styling-appearance-glyphs-reference-sheet%}) article.
+
+## Properties
+
+The RadGlyph class exposes the following properties:
+
+* **Glyph**: The actual text to be displayed. It is of type **String**.
+
+* **Foreground**: The foreground of the element which would color the rendered glyph. It is of type **Brush**.
+
+* **Background**: The background of the element. It is of type **Brush**.
+
+* **Font**: The name of the font to be used. It is of type **String** and corresponds to one of the [registered fonts](#register-fonts).
+
+* **FontSize**: The font size of the element. It is of type **Double**.
+
+The RadGlyphExtension on the other hand provides this set of properties:
+
+* **Glyph**: The actual text to be displayed. It is of type **Object**.
+
+* **Foreground**: The foreground of the element which would color the rendered glyph. It is of type **Brush**.
+
+* **Font**: The name of the font to be used. It is of type **String** and corresponds to one of the [registered fonts](#register-fonts).
+
+* **Size**: The font size of the element. It is of type **Double**.
+
+* **Type**: Explicitly sets type of the exported glyph. It is of type **GlyphExportType** which is an enumeration with the values **None** (default), **Image** (which returns an **ImageSource**) and **Path** (which returns a **Geometry**).
+
+## Register Fonts
+
+The default font family used by the glyphs is the **TelerikWebUI**. A list of the available glyphs can be found in the [Glyphs Reference Sheet]({%slug common-styling-appearance-glyphs-reference-sheet%}) article.
+
+If you wish to use a custom font in your project, you first have to register it by calling the static **RadGlyph.RegisterFont** method. It has the following 4 overloads:
+
+* **RegisterFont(string fontName)**
+* **RegisterFont(FontFamily fontFamily)**
+* **RegisterFont(FontFamily fontFamily, string fontName)**
+* **RegisterFont(Uri fontUri, string fontName)**
+
+**Example 3** shows how to register the **Webdings** typeface, which is included in all versions of Microsoft Windows **since Windows 98**.
```

Comment:
We can add a link to the Webdings wiki page: https://en.wikipedia.org/wiki/Webdings